### DISCUSSION TITLE
Unlink Game Entry Changes

### DISCUSSION THREAD

**Discussion by kdecks** [RetroAchievements/RAWeb#660] [Ideas]
1 Add a field where developers can label each MD5.
2 Add a CRC32 field. Hopefully this is generated automatically.
3. Change the name of **Unlink Game Entry** to Linked Version Manager.

See: #1 Show Linked hashes on game page

**Comment by Jamiras** [comment]
Love the idea of labelling MD5s. I think #47 should have superceded this instead of the other way around. Maybe update the title to be similar to #47.

**Comment by meleu** [comment]
Related: #717

**Comment by Jamiras** [comment]
Marking complete (can't close discussions) as this is better handled by #717.
